core/project_settings.cpp

```diff
@@ -742,6 +755,26 @@ Error ProjectSettings::_save_custom_bnd(const String &p_file) { // add other par
 	return save_custom(p_file);
 };
 
+bool ProjectSettings::is_settings_text_up_to_date_custom(const String &p_path) {
+
+	print_line(p_path);
+	if (!p_path.ends_with(".godot")) {
+		ERR_EXPLAIN(String("Invalid project settings text file (should end in .godot) : ") + p_path);
```

Comment:
Should not have a space before ":" in English.

But is there any case where we check a `.godot` file which is not `project.godot`? The only other kind of settings file would be `override.cfg` I think.

Reply:
Indeed, the test makes little sense, I'll remove it.